Formula/nvc.rb

```diff
@@ -19,8 +28,10 @@ class Nvc < Formula
   end
 
   depends_on "check" => :build
-  depends_on "llvm" => :build
   depends_on "pkg-config" => :build
+  # llvm 8+ is not supported https://github.com/nickg/nvc/commit/c3d1ae5700cfba6070293ad1bb5a6c198c631195
+  # and llvm 7 has issue on stable https://github.com/nickg/nvc/commit/dfd5606a182e00d5f7a9e28902234b374c7b2863
+  depends_on "llvm@6"
```

Comment:
Can we add a patch from https://github.com/nickg/nvc/commit/dfd5606a182e00d5f7a9e28902234b374c7b2863 and use `llvm@7`?

Reply:
i can try it

Reply:
the patch does not seems work https://github.com/Homebrew/homebrew-core/runs/685654208. that's why I had to use llvm@6. lmk any thought

Reply:
bump. any thought?